### ISSUE TITLE
DELETE method not working

### ISSUE THREAD

**Issue by miguki** [jparis/qrs-interact#70]
req.end();

is not called in DELETE method definition, so request is not taking place

**Comment by miguki** [comment]
```
this.Delete = function(path) {
        return new Promise(function(resolve, reject) {
            var r = requestDefaultParams;
            path = getFullPath(path);
            r['method'] = 'DELETE';
            r['path'] = path;

            var req = https.request(r, (res) => {
                var statusCode = res.statusCode;
                res.on('end', function() {
                        if (statusCode == 204) {
                            resolve(statusCode);
                        } else {
                            reject("Received error code: " + statusCode);
                        }
                    })
                    .on('error', function(err) {
                        reject("QRS response error:" + err);
                    });
            }).on('error', function(err) {
                reject("QRS request error:" + err);
            });
        });
    };
```

**Comment by jparis** [comment]
Oops. typo when I updated the code. Clearly don't have any tests checking deletes. Will update and also add some tests. Thanks.

**Comment by miguki** [comment]
Also in DELETE method https response events are not fired, so promises cannot be resolved

**Comment by jparis** [comment]
fixed in 6.1.0
